source/lib/course-search/index.js

```diff
@@ -4,3 +4,4 @@ export {loadCachedCourses} from './load-cached-courses'
 export {updateStoredCourses, areAnyTermsCached} from './update-course-storage'
 export {CourseType, TermType} from './types'
 export {parseTerm} from './parse-term'
+export {loadGEs} from './load-ges'
```

Comment:
Where is this called?

Reply:
Looks to have been introduced in https://github.com/StoDevX/AAO-React-Native/pull/2378/commits/d2fdcdbf6c3b4f2c15b449b8cad5f7df607f99e1#diff-df4b353eb39c9a68e6b0e3127a055bb4R69
and removed in https://github.com/StoDevX/AAO-React-Native/pull/2378/commits/da79336f7c854353c356e911d0e4650737a8c629#diff-df4b353eb39c9a68e6b0e3127a055bb4L84

Reply:
Yeah that was left over from my first attempt when I wasn't reusing the filter component..I kept it so that I could ask this question: For access to the acceptable list of GEs, departments, etc. should we load that data from the server and store it just like the course data, or should we try to somehow extract them from the course data itself?

Reply:
I'd recommend using https://github.com/StoDevX/course-data/blob/gh-pages/data-lists/valid_gereqs.json for a list of GE requirements.

It's generated automatically from the course data every night.

Of, you can do something to determine the gereqs from the downloaded course data; that's what I do in Gobbldygook, too.

Reply:
This PR fetches https://stodevx.github.io/course-data/data-lists/valid_gereqs.json

Reply:
That's what it uses currently.